src/frontend/src/pages/Index/Settings/AdminCenter/PartParameterPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import { t } from '@lingui/macro';
+import { Accordion } from '@mantine/core';
+
+import { StylishText } from '../../../../components/items/StylishText';
+import PartParameterTemplateTable from '../../../../tables/part/PartParameterTemplateTable';
+import SelectionListTable from '../../../../tables/part/SelectionListTable';
+
+export default function PartParameterPanel() {
+  return (
+    <>
+      <PartParameterTemplateTable />
```

Comment:
Can you move the `<PartParameterTemplateTable />` down into the accordion below (default selected) - to match UX for other panels where there are multiple tables

Reply:
fixed